Python/2. variables.md

```diff
@@ -46,7 +53,9 @@ Dictionary: {"what" : 3, "blah" : "oh no"}
 List: [3, 4, "D", 2.4, True, False, [[3, 4, 5, 6, 7], True]]
 ```
 
-Note: `\n` means a new line. If you add that in the middle of a print statement in double or single quotes, it will print the rest on a new line.
+`\n` means a new line. If you add that in the middle of a print statement in double or single quotes, it will print the rest on a new line. You can also add newlines inside of lists, dictionaries, and tuples `()`. To check the type of a variable, enter `>>> type(aVar)` or `print(type(aVar))`.
```

Comment:
again, don't you mean `>>> print(type(aVar))'?

P.S. What is the reason for these >>>s. If you write them in the code it will be an error right

Reply:
The `>>>` means running the code through a REPL, not a `.py` file. (Type `python` in the shell)

Reply:
ok. But do you think we should use the `>>>` or not. I added a commit with most of the things you added, except the `>>>`.